src/code/z_camera.c

```diff
@@ -6956,9 +6990,9 @@ void Camera_Init(Camera* camera, View* view, CollisionContext* colCtx, PlayState
     camera->up.z = 0.0f;
     camera->fov = 60.0f;
     camera->pitchUpdateRateInv = R_CAM_DEFAULT_PITCH_UPDATE_RATE_INV;
-    camera->xzOffsetUpdateRate = CAM_DATA_SCALED(OREG(2));
-    camera->yOffsetUpdateRate = CAM_DATA_SCALED(OREG(3));
-    camera->fovUpdateRate = CAM_DATA_SCALED(OREG(4));
+    camera->xzOffsetUpdateRate = CAM_DATA_SCALED(R_CAM_XZ_UPDATE_RATE);
+    camera->yOffsetUpdateRate = CAM_DATA_SCALED(R_CAM_Y_OFFSET_UPDATE_RATE);
```

Comment:
You have the "offset" for y but missing for xz
```suggestion
    camera->xzOffsetUpdateRate = CAM_DATA_SCALED(R_CAM_XZ_OFFSET_UPDATE_RATE);
    camera->yOffsetUpdateRate = CAM_DATA_SCALED(R_CAM_Y_OFFSET_UPDATE_RATE);
```